### PR TITLE
add MPS agg_mps adapter 

### PR DESCRIPTION
**Pull request by mbkranz**
Jira Ticket: [HP-523](https://ctds-planx.atlassian.net/browse/HP-523)

This pull request adds an adapter to the [Microphysiological Systems Database studies API](https://mps.csb.pitt.edu/api/studies/).

### CODE REVIEW

File: src/mds/agg_mds/adapters.py [hunk 1]
```diff
@@ -177,6 +178,102 @@ def getMetadata(self, **kwargs):
...
+
+        if len(study_ids) > 0:
+            for id in study_ids:
+                try:
+                    #get url request put data into datadict
+                    url = f"{mds_url}/{id}/"
+                    response = httpx.get(url)
+                    response.raise_for_status()
+
+                    data_dict = json.loads(response.text)
```
Comment (mfshao) [inline src/mds/agg_mds/adapters.py:214]: I think you can use `response.json()` instead of importing `json` directly

**Review by mfshao — COMMENTED** [review]
overall lgtm, one small comment. I'll do the work to run CI on this PR soon

also there are some syntax/format issues that `black` is not happy with, you should fix them

File: src/mds/agg_mds/adapters.py (hunk 1 above)
Comment (mbkranz) [reply src/mds/agg_mds/adapters.py]: @mfshao  You're right -- just changed. Is this ready for a merge or do you want me to do something with the "in development" test script before ready? I was thinking -- for now -- may be best to keep separate from test/agg_mds_adapters.py given how long it is.

File: tests/test_agg_mds_mps_repo.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            content=json_response,
+        )
+        
+    assert get_metadata(
+            "mps",
+            "http://test/ok",
+            filters={"study_ids": [23]},
+            mappings=field_mappings,
+            keepOriginalFields=True,
+        )=={
```
Comment (mfshao) [inline tests/test_agg_mds_mps_repo.py:100]: the assertion result on the right should also have `tags`?

Comment (mbkranz) [reply tests/test_agg_mds_mps_repo.py]: hey @mfshao  the assertion had a tag but wasn't formatted --- I formatted code with black and tag should be easier to see I think?

Comment (mbkranz) [reply tests/test_agg_mds_mps_repo.py]: @mfshao Hey - am would it be possible to run the testing to see if this passes? (see above comment)

**Comment by mbkranz** [comment]
Opened another PR (#62) to finish up the testing

**Pull request closed without merging**